ui/StatusQ/include/StatusQ/formatteddoubleproperty.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <QmlTypeAndRevisionsRegistration>
+
+class FormattedDoubleProperty : public QObject
```

Comment:
Imo this component requires some basic documentation describing what's the purpose and behavior of that component.
At least at first glance that component does the work that can be easily done in qml without instantiating additional QObject for that (var property for value and readonly string property for parsed string representation).

Reply:
Unit tests should be added. They would also nicely show corner cases behavior.

Reply:
Agree, I'll add something